### ISSUE TITLE
stanheader src directory not in quotes with current develop branch 

### ISSUE THREAD

**Issue by maverickg** [stan-dev/rstan#170]
Because the path is not put in quotes, the space is going to be an issue. 

I will fix it. 

```
> install.packages('rstan_2.7.0.tar.gz',type='source', repos=NULL)
* installing *source* package 'rstan' ...
** package 'rstan' successfully unpacked and MD5 sums checked
** libs
g++ -m64 -I"C:/PROGRA~1/R/R-31~1.3/include" -DNDEBUG -I"../inst/include" -I`"C:/PROGRA~1/R/R-31~1.3/bin/x64/Rscript" -e "cat(system.file('include', 'src', package = 'StanHeaders'))"` -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS   -I"C:/Program Files/R/R-3.1.3/library/Rcpp/include" -I"C:/Program Files/R/R-3.1.3/library/RcppEigen/include" -I"C:/Program Files/R/R-3.1.3/library/BH/include" -I"C:/Program Files/R/R-3.1.3/library/StanHeaders/include" -I"d:/RCompile/CRANpkg/extralibs64/local/include"     -O2 -Wall  -mtune=core2 -c chains.cpp -o chains.o
g++.exe: error: Files/R/R-3.1.3/library/StanHeaders/include/src: No such file or directory
make: *** [chains.o] Error 1
Warning: running command 'make -f "Makevars.win" -f "C:/PROGRA~1/R/R-31~1.3/etc/x64/Makeconf" -f "C:/PROGRA~1/R/R-31~1.3/share/make/winshlib.mk" SHLIB_LDFLAGS='$(SHLIB_CXXLDFLAGS)' SHLIB_LD='$(SHLIB_CXXLD)' SHLIB="rstan.dll" WIN=64 TCLBIN=64 OBJECTS="chains.o init.o lang__ast_def.o lang__grammars__bare_type_grammar_inst.o lang__grammars__expression07_grammar_inst.o lang__grammars__expression_grammar_inst.o lang__grammars__functions_grammar_inst.o lang__grammars__program_grammar_inst.o lang__grammars__statement_2_grammar_inst.o lang__grammars__statement_grammar_inst.o lang__grammars__term_grammar_inst.o lang__grammars__var_deccls_grammar_inst.o lang__grammars__whitespace_grammar_inst.o misc.o stanc.o"' had status 2
ERROR: compilation failed for package 'rstan'
* removing 'C:/Program Files/R/R-3.1.3/library/rstan'
Warning messages:
1: running command '"C:/PROGRA~1/R/R-31~1.3/bin/x64/R" CMD INSTALL -l "C:\Program Files\R\R-3.1.3\library" "rstan_2.7.0.tar.gz"' had status 1 
2: In install.packages("rstan_2.7.0.tar.gz", type = "source", repos = NULL) :
  installation of package ‘rstan_2.7.0.tar.gz’ had non-zero exit status
```

**Comment by bgoodri** [comment]
This creates a portability warning from R CMD check.

**Comment by bgoodri** [comment]
Maybe the old way would have worked with `format(system.file('include', 'src', package = 'StanHeaders'))`?

**Comment by maverickg** [comment]
I think it's fixed with 6f9542c5bf008d583b2ae290cd3cae69bc5bd4d5
